src/service/server/LoadHandler.cpp

```diff
@@ -201,7 +201,9 @@ LoadResult LoadHandler::loadGoogleSheet(const LoadRequest* req, LoadError& err,
   // by comparing the json value in table settings
   rapidjson::Document doc;
   if (doc.Parse(json.c_str()).HasParseError()) {
-    throw NException(fmt::format("Error parsing google sheet json: {0}", json));
```

Comment:
Curious, how did you hit this? this is used to load a google spreadsheet through API

Reply:
@shawncao I don't hit this - I just get an arg not used build warning due to `err` so I figured I might as well use it like this to silence the warning and make it behave the same as the rest